ql/src/test/queries/clientpositive/cbo_rp_null_order.q

```diff
@@ -0,0 +1,7 @@
+SET hive.cbo.returnpath.hiveop=true;
+SET hive.default.nulls.last=false;
```

Comment:
For completeness maybe we want to add an extra test with `nulls.last=true` to ensure that changes take effect.

Reply:
We have similar tests but not with this query so I added.